src/sentry/integrations/github/webhook.py

```diff
@@ -71,11 +71,15 @@
 logger = logging.getLogger("sentry.webhooks")
 
 
+# Functions that process webhook events need to have this signature.
+# This is used to type check the webhook processors.
 class WebhookProcessor(Protocol):
     def __call__(
         self,
         *,
-        event_type: str,
+        # This comes from the X-GitHub-Event header
+        github_event: GithubWebhookType,
```

Comment:
Renaming the variable to make it dead obvious. `event_type` refers to an internal integrations metric.